### PR TITLE
🚑  Fix empty data in CosmosDB

### PR DESCRIPTION
**Pull request by bradystroud**
Do proper data validation

### CODE REVIEW

File: src/services/apiService.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ export async function GetAllLikedDisliked(userId) {
 }
 
 export async function PostReactionForUser(data, token) {
-  if (data == null) {
+  const isEmpty = Object.values(data).some((x) => x == null || x == '');
+  if (data == null || isEmpty) {
```
Comment (tombui99) [inline src/services/apiService.js:22]: use "!data" instead of "data == null"

File: src/services/apiService.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ export async function GetAllLikedDisliked(userId) {
 }
 
 export async function PostReactionForUser(data, token) {
-  if (data == null) {
+  const isEmpty = Object.values(data).some((x) => x == null || x == '');
```
Comment (tombui99) [inline src/services/apiService.js:21]: Do this instead
const isEmpty = Object.values(data).some((x) => !x);

Comment (bradystroud) [reply src/services/apiService.js:21]: Will that check if x is an empty string?

Comment (tombui99) [reply src/services/apiService.js]: Yes it does
https://stackoverflow.com/questions/6003884/how-do-i-check-for-null-values-in-javascript

Comment (bradystroud) [reply src/services/apiService.js]: Ahh cool, I didn't know that, Thanks!

**Review by tombui99 — APPROVED** [review]
LGTM

**Pull request merged**